homeassistant/components/modbus/switch.py

```diff
@@ -209,7 +208,6 @@ def turn_on(self, **kwargs):
 
     def turn_off(self, **kwargs):
         """Set switch off."""
-
```

Comment:
This has nothing to do with changing constants (which I see as a good idea), the empty line is design,  please remove this change.